UIAComWrapper.proj

```diff
@@ -28,23 +28,12 @@
   <Target Name="Build" DependsOnTargets="Cleanup">
     <MSBuild Projects="$(Root)UIAComWrapper.sln" Properties="Configuration=$(Configuration);Platform=$(Platform)" />
   </Target>
-
-  <Target Name="Test" DependsOnTargets="Build">
-    <ItemGroup>
-      <TestAssembly Include="$(Root)UIAComWrapperTests\bin\$(Configuration)\UIAComWrapperTests.dll"/>
-      <TestResults Include="$(Root)UIAComWrapperTests\bin\$(Configuration)\UIAComWrapperTests.trx"/>
-    </ItemGroup>
-
-    <PropertyGroup >
-      <VisualStudioDir>C:\Program Files (x86)\Microsoft Visual Studio 11.0\</VisualStudioDir>
-      <MSTest>$(VisualStudioDir)Common7\IDE\MSTest.exe</MSTest>
-    </PropertyGroup>
-
-    <Exec Command='"$(MSTest)" /testcontainer:@(TestAssembly) /resultsfile:@(TestResults)'/>
-    <Message Text='##teamcity[importData type="mstest" path="@(TestResults)"]'/>
-  </Target>
   
-  <Target Name="Publish">
+  <Target Name="IndexSources" DependsOnTargets="Build">
+    <Exec Command='tools\GitLink\GitLink.exe $(Root) -u https://github.com/TestStack/UIAComWrapper -c $(Configuration)' />
```

Comment:
Note that you should execute that command line - either via batch file or manually... I don't know exact process of publishing nuget package, so left that as batch-file-only line. Feel free to make it convenient as you see fit.